src/components/overlays/pause.js

```diff
@@ -17,9 +17,11 @@ export class Pause extends Screen {
     create() {
         this.addBackgroundItems();
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
+        const isAboveSelectScreen = parentKey.includes("select");
```

Comment:
Do we only want it to show if you are not on _any_ select screen? i.e you could be on 3 of 5 select screens and you won't see the button to take you back to the first select screen?

Reply:
saw this now, was thinking the same thing when I commented 😛

Reply:
I see the ticket suggests that the play button is for that purpose, I guess it may be confusing if you see a select screen button while you're on a select screen and it takes you to a different select screen (the first one)

Reply:
Yeah it is a valid point but the ticket says dont show on select, plus this button itself has only existed for like two weeks and isnt in any game yet so its not like lost functionality